doc/classes/EditorExportPlatformExtension.xml

```diff
@@ -36,7 +36,21 @@
 			<description>
 				[b]Optional.[/b]
 				Creates a PCK archive at [param path] for the specified [param preset].
-				This method is called when "Export PCK/ZIP" button is pressed in the export dialog, and PCK is selected as a file type.
+				This method is called when "Export PCK/ZIP" button is pressed in the export dialog, with "Export as Patch" disabled, and PCK is selected as a file type.
+			</description>
+		</method>
+		<method name="_export_pack_patch" qualifiers="virtual">
+			<return type="int" enum="Error" />
+			<param index="0" name="preset" type="EditorExportPreset" />
+			<param index="1" name="debug" type="bool" />
+			<param index="2" name="path" type="String" />
+			<param index="3" name="patches" type="PackedStringArray" />
+			<param index="4" name="flags" type="int" enum="EditorExportPlatform.DebugFlags" is_bitfield="true" />
+			<description>
+				[b]Optional.[/b]
+				Creates a patch PCK archive at [param path] for the specified [param preset], containing only the files that have changed since the last patch.
+				This method is called when "Export PCK/ZIP" button is pressed in the export dialog, with "Export as Patch" enabled, and PCK is selected as a file type.
+				[b]Note:[/b] [param patches] serves as an optional override for the patches defined in the export preset. When empty the patches defined in the export preset will be used instead.
```

Comment:
```suggestion
				[b]Note:[/b] [param patches] serves as an optional override for the patches defined in the export preset. When empty the patches defined in the export preset should be used instead.
```
Assuming as this is a virtual method it should describe what to do with this method

Reply:
So this is a bit of a gotcha I suppose, but the patches are actually loaded/unloaded before/after this virtual method is invoked. I struggled to see the use-case for having the extension class do anything differently here, so in order to avoid needing to expose some `load_patches` and `unload_patches` methods on `EditorExportPlatform`, that the extension class would invoke itself, I opted to instead just do it behind the scenes.

The `patches` parameter itself is borderline useless to the extension class as things stand now, but it's an important part of the context so I should still pass it in.

I can change this if it's too weird, or I can make it clearer in the documentation that this is what happens.

Reply:
I'd say it should be clear how this should be used by the virtual method, as this wording is more about how the base method will use it, I'd this argument isn't something that can change anything I'd say it's weird to pass it here and instead the actual values should be passed based on either the passed argument or the default

Reply:
I.e. either this should be clarified or changed to "should" to communicate that it is the responsibility of who ever implements this method to carry this selection out

Reply:
How about this?

```suggestion
				[b]Note:[/b] The patches provided in [param patches] have already been loaded when this method is called and are merely provided as context. When empty the patches defined in the export preset have been loaded instead.
```